rts/lra/client/src/main/java/io/narayana/lra/client/NarayanaLRAClient.java

```diff
@@ -139,7 +144,7 @@
      * If not defined as default value is taken {@code http://localhost:8080/lra-coordinator}.
      * The LRA recovery coordinator will be searched at the sub-path {@value LRAConstants#RECOVERY_COORDINATOR_PATH_NAME}.
      *
-     * @throws IllegalStateException  thrown when the URL taken from the system property value is not an URI format
+     * @throws IllegalStateException  thrown when the URL taken from the system property value is not a URI format
```

Comment:
agree, just I'm not sure how to change it. We use the term `coordinatorUrl` and expecting the URL in the code but the internal format is URI.
Should that be `thrown when the URI taken from the system property value is not a URI format`?

Reply:
A URL is still a URI, or you could change the text URL to URI, or, better, you could insist that it's a URL since that is the only format we support for coordinator endpoints. TBH we should have stuck with URL (since that is that is the only way to access the resources) in the spec but no big deal, as long as we are consistent in our wording.

Reply:
Ok, changed to URL.